src/System.ServiceModel.Federation/src/System/ServiceModel/Federation/WSTrustChannelFactory.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+using System.ServiceModel.Channels;
+using System.ServiceModel.Description;
+using Microsoft.IdentityModel.Protocols.WsTrust;
+
+namespace System.ServiceModel.Federation
+{
+    /// <summary>
+    /// A <see cref="WSTrustChannelFactory" /> that produces <see cref="WSTrustChannel" /> objects used to communicate with a WS-Trust endpoint.
+    /// </summary>
+    [ComVisible(false)]
+    public class WSTrustChannelFactory : ChannelFactory<IWSTrustChannelContract>
```

Comment:
I think this can probably be removed. It looks like you added it because originally on NetFx this was there.

Reply:
addressed in: ab9b3c1cb00f558dcd6ac6f3b02663ace5e25750